packages/@sanity/base/src/components/formField/FormFieldSet.tsx

```diff
@@ -127,7 +127,7 @@ export const FormFieldSet = forwardRef(
         return null
       }
       return (
-        <Grid columns={columns} gapX={4} gapY={5}>
+        <Grid columns={columns} gapX={4} gapY={5} style={{alignItems: 'flex-end'}}>
```

Comment:
Nice!

I would suggest not using inline styling, but instead create a styled component of the `Grid` primitive outside the component scope.

```js
const StyledGrid = styled(Grid)`
  align-items: flex-end;
`
```